engines/ultima/shared/gfx/sprites.cpp

```diff
@@ -52,6 +52,13 @@ Sprite::Sprite(const byte *src, uint bpp, uint16 w, uint16 h) {
 	}
 }
 
+Sprite &Sprite::operator=(const Sprite &src)
```

Comment:
It's a minor nitpick, but our style guide would have the { on the same line as the method, as in (const Sprite &src) {

Reply:
Will amend.

Reply:
Done